lib/elastic_record/version.rb

```diff
@@ -0,0 +1,11 @@
+module ElasticRecord
+  class Version
+    include Model
+
+    def self.es6?
+      @es6_mode ||= elastic_connection.json_get('/')
+        .dig('version', 'number')
+        .start_with?('6.')
```

Comment:
I think you want a safe navigation operator if you're going to be `.dig`ging.

```suggestion
        &.start_with?('6.')
```